catkit2/services/thorlabs_cubes/thorlabs_cubes.py

```diff
@@ -0,0 +1,108 @@
+from catkit2.testbed.service import Service
+
+import thorlabs_apt as apt
+import numpy as np
+
+
+class ThorlabsCubes(Service):
+
+    def __init__(self):
+        super().__init__('thorlabs_cubes')
+
+        self.cube_serial_number = self.config['serial_number']
+
+        self.command = self.make_data_stream('command', 'float64', [1], 20)
+        self.current_position = self.make_data_stream('current_position', 'float64', [1], 20)
+
+        # Initialization of the parameters

```

Comment:
Either make this comment more explicit or delete it - setting a bunch of variables to `None` is very obviously an initialization (commenting code is always good, especially while developing, but catkit2 has a minimal comment approach where we try to omit comments that are too obvious. It's a choice, not a rule.).